.azure-pipelines/validate-pr-auth-module.yml

```diff
@@ -14,8 +14,8 @@ pr:
 trigger: none
 
 jobs:
-- job: MSGraphPSSDKValidation
-  displayName: MS Graph PS SDK Auth Validation
+- job: MSGraphPSSDKValidation_Windows
+  displayName: MS Graph PS SDK Auth Validation - Windows
```

Comment:
We should move this off of the Microsoft Graph pool. We should only use this pool for pipeline debugging/development.